src/Nordea/Components/Label.elm

```diff
@@ -216,7 +216,7 @@ stateStyles { hasError } =
 
         outlineStyle =
             Css.batch
-                [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ color) |> Css.important
```

Comment:
scary @fredrikloberg solution 👀 😅

Reply:
I would also remove the two importants on line 225 and 226 just below while you are at it